docs/js-client-sdk.eppojsclient.getexperimentcontainerentry.md

```diff
@@ -0,0 +1,24 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->

```

Comment:
The experiment container registry docs must have been missed in a previous commit?

Reply:
🤷‍♂️ 

I hate these doc generators 😞